master/reference/public-cloud/aws.md

```diff
@@ -20,13 +20,13 @@ on your deployment.
 Calico requires the following security group exceptions to function properly
 in AWS.
 
-| Description      | Type            | Protocol | Port Range |
-|:-----------------|:----------------|:---------|:-----------|
-| BGP              | Custom TCP Rule | TCP      | 179        |
-| \*IPIP           | Custom Protocol | IPIP     | all        |
+| Description        | Type            | Protocol | Port Range |
+|:-------------------|:----------------|:---------|:-----------|
+| `BGP`              | Custom TCP Rule | TCP      | 179        |
```

Comment:
What's the reasoning for code ticks here? They're just acronyms. We don't seem to do it to the ones in the protocol column

Reply:
This is a good point. It seems that they need to select a specific option from the AWS UI in which case it should have backticks or bolding, but without an AWS instance, it's hard for me to confirm this. When I looked at this note, it had `\*IPIP` for some reason. I had thought that this was a real option within AWS but I am thinking more and more that this might have been a typo. Let me look into this further.

Reply:
The `*` asterisks is for the note four lines down